modules/wechat_qrcode/src/decodermgr.cpp

```diff
@@ -46,6 +46,10 @@ int DecoderMgr::decodeImage(cv::Mat src, bool use_nn_detector, string& result) {
         int ret = TryDecode(source, zx_result);
         if (!ret) {
             result = zx_result->getText()->getText();
+            auto result_points = zx_result->getResultPoints();
+            for(int i=0; i < result_points->size(); i++){
```

Comment:
Corners should be order as Top Left -> Top Right -> Bottom Right -> Bottom Left
`result_points` are returned in Bottom Left -> Top Left -> Top Right -> Bottom Right